.github/workflows/coverage.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Code Coverage
+
+on:
+  push:
+    branches-ignore: dev/*
+  pull_request:
+
+jobs:
+  Coveralls:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm ci
+    - run: npx jest --coverage
```

Comment:
For our PRs, will this cause us to be running through the unit tests twice?

Reply:
yeah it will run our unit tests one extra time. Currently they are run for 6 different permutations, with 3 different node versions, and the git branch being either the pushed branch, or the result of the pushed branch being merged with the base branch. I can look into trying to combine this with our run-tests.yml